pkg/tcl/testworkflowstcl/testworkflowprocessor/container.go

```diff
@@ -290,6 +289,10 @@ func (c *container) ToContainerConfig() testworkflowsv1.ContainerConfig {
 	for i := range envFrom {
 		envFrom[i] = *envFrom[i].DeepCopy()
 	}
+	volumeMounts := slices.Clone(c.VolumeMounts())
+	for i := range volumeMounts {
+		volumeMounts[i] = *volumeMounts[i].DeepCopy()
```

Comment:
looks a bit weird comparing to previoua code

Reply:
The previous code was [**the same**](https://github.com/kubeshop/testkube/pull/5158/files#diff-9defc5a48f14e78f1599c36e62547326d6143416c7334ba5f344f0cfea3f5152L309-L316) 🙂